src/renderer/Components/RecevingPackage/index.jsx

```diff
@@ -57,22 +111,26 @@ function ReceivingPackage() {
           일련번호
         </label>
         <div className="flex justify-center">
-          {Array(6).fill(
-            <NumberInput
-              validateNumber={validateNumber}
-              handleMoveNextInput={handleMoveNextInput}
-            />,
-          )}
+          {Array(6)
+            .fill()
+            .map((_, index) => (
+              <NumberInput
+                key={index}
+                validateNumber={() => validateNumber(event, index)}
+                handleMoveNextInput={handleMoveNextInput}
```

Comment:
```suggestion
                handleFocusShift={handleFocusShift}
```
여기 수정사항은 아니지만 지금보니 '인풋 포커스를 옮기는 것을 또 핸들링한다' 는 뉘앙스라 어색한것 같네요. 이 제안은 어떨까요?